### PR TITLE
Remove --hyperkube and KUBE_BUILD_HYPERKUBE references

### PR DESCRIPTION
**Pull request by dims**
follow up from change in k/k where we removed hyperkube, we should fix
up the jobs and scripts to remove any references.

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

File: scenarios/kubernetes_build.py [hunk 1]
```diff
@@ -120,8 +120,6 @@ def main(args):
         push_build_args.append('--bucket=%s' % args.release)
     if args.registry:
         push_build_args.append('--docker-registry=%s' % args.registry)
-    if args.hyperkube:
-        env['KUBE_BUILD_HYPERKUBE'] = 'y'
```
Comment (BenTheElder) [inline scenarios/kubernetes_build.py]: are we using this on older branches?

Comment (dims) [reply scenarios/kubernetes_build.py]: the CI jobs didn't seem to need it when i eye-balled them.

**Comment by justaugustus** [comment]
LGTM, but:
/hold until 1.18.0 is released

**Comment by dims** [comment]
we are past 1.18.0
/hold cancel

/assign @spiffxp

**Comment by spiffxp** [comment]
hyperkube is still referenced in a few kubetest providers, and two jobs
```
$ ag -l hyperkube
kubetest/local.go
kubetest/azure_helpers.go
kubetest/kubeadmdind/kubeadm_dind.go
kubetest/azure_test.go
kubetest/azure.go
config/jobs/kubernetes/sig-testing/coverage.yaml
config/jobs/kubernetes/sig-testing/local-e2e.yaml
```
Looking to see what can / can't be removed

**Comment by spiffxp** [comment]
Remove from:
- kubetest/local.go
- kubetest/kubeadmdind/kubeadm_dind.go
- config/jobs/kubernetes/sig-testing/coverage.yaml
- config/jobs/kubernetes/sig-testing/local-e2e.yaml

This PR is a bit out of date, azure is now aksengine, and still needs hyperkube for < 1.17

**Comment by dims** [comment]
@spiffxp DONE (removed references from local.go and the 2 yaml files)

**Pull request merged**